internal/app/singularity/cache_list_linux.go

```diff
@@ -16,6 +16,9 @@ import (
 	"github.com/sylabs/singularity/internal/pkg/sylog"
 )
 
+// findSize will take a int64 (like: '4781321234') and convert it to human readable output
+// in Kb, Mb, and Gb. output will be a string.
```

Comment:
since these are bytes and not bits (I assume), the SI units are "kB", "MB" and "GB". It's probably not a bad idea to add "TB" to the list, too.

And to prevent other people from having to make an assumption, it's probably a good idea to document that the input is bytes, so you could change the text to something along the lines of "findSize takes a size in bytes and converts it to a human-readable string representation expressing kB, MB or GB (whatever is smaller, but still larger than one)."

Also note that the actual code is using "Kb", "Mb", "Gb", so that should be fixed as well.